codegen/codegen.cabal

```diff
@@ -0,0 +1,23 @@
+cabal-version:       2.2
+name:                codegen
+version:             0.1.0.0
+-- synopsis:
+-- description:
+homepage:            https://github.com/patrickt/codegen#readme
+license:             BSD-3-Clause
+author:              Patrick Thomson
+maintainer:          patrickt@github.com
+copyright:           Patrick Thomson
+category:            Web
```

Comment:
Some things to fix here:

- homepage is wrong
- author should definitely be @aymannadeem, this is her work
- maintainer I’m not sure about? Do we have a good email address to use for this rather than using our individual work accounts? cc @tree-sitter/haskell for thoughts.
- copyright should be GitHub; not sure if it should mention the year as well?

Reply:
You can use `opensource+semantic@github.com` for the maintainer address, that will be what we use for semantic itself.